modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/attributes/AccountsAttributesModal.tsx

```diff
@@ -14,36 +14,43 @@
 
 import React from 'react';
 
+import {fetchAccountsFields, updateAccountsFields} from '../../utils/api';
 import Modal, {ICommonModalProps} from './Modal';
 
+const columns = [
+	{
+		expanded: true,
+		label: Liferay.Language.get('attribute'),
+		value: 'attribute',
+	},
+
+	{
+		expanded: true,
+		label: Liferay.Language.get('data-type'),
+		value: 'dataType',
+	},
+	{
+		expanded: true,
+		label: Liferay.Language.get('sample-data'),
+		value: 'sampleData',
+	},
+];
+
```

Comment:
Hi @ilzamcmed, instead create a column for accounts, please move it from `PeopleAttributesModal` to inside this component https://github.com/liferay/liferay-portal/blob/master/modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/components/attributes/Modal.tsx and all components that call the Modal should use the same column.

Reply:
Fixed